tests/Jenkins/dirac-cfg-update.py

```diff
@@ -34,7 +40,16 @@
 if cFile:
   localConfigFile = cFile
 else:
-  localConfigFile = './etc/dirac.cfg'
+  print "WORKSPACE: %s" % os.path.expandvars('$WORKSPACE')
+  if os.path.isfile( os.path.expandvars('$WORKSPACE')+'/PilotInstallDIR/etc/dirac.cfg' ):
+    localConfigFile = os.path.expandvars('$WORKSPACE')+'/PilotInstallDIR/etc/dirac.cfg'
+  elif os.path.isfile( os.path.expandvars('$WORKSPACE')+'/ServerInstallDIR/etc/dirac.cfg' ):
+    localConfigFile = os.path.expandvars('$WORKSPACE')+'/ServerInstallDIR/etc/dirac.cfg'
+  elif os.path.isfile( './etc/dirac.cfg' ):
+    localConfigFile = './etc/dirac.cfg'
+  else:
```

Comment:
This check of a series of paths in the configuration in order is done in multiple places. Maybe it is useful to create a utility to do this for everyone?